wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -369,6 +369,46 @@ void raceWithOrderTest() {
     }
   }
 
+  @Test
+  void repeatedlyTest() {
+    try (CommandScheduler scheduler = new CommandScheduler()) {
+      AtomicInteger counter = new AtomicInteger(0);
+
+      Command command = new InstantCommand(counter::incrementAndGet);
+
+      Command group = command.repeatedly();
+
+      scheduler.schedule(group);
+      for (int i = 1; i <= 50; i++) {
+        scheduler.run();
+        assertEquals(i, counter.get());
+      }
+
+      // Should still be scheduled
+      assertTrue(scheduler.isScheduled(group));
+    }
+  }
+
+  @Test
+  void repeatForTest() {
+    try (CommandScheduler scheduler = new CommandScheduler()) {
+      AtomicInteger counter = new AtomicInteger(0);
+
+      Command command = new InstantCommand(counter::incrementAndGet);
+
+      Command group = command.repeatedly(3);
+
+      scheduler.schedule(group);
+      for (int i = 0;
+          scheduler.isScheduled(group);
+          i++) { // If this causes an infinite loop, repeatedly is cooked
+        scheduler.run();
+        assertEquals(i + 1, counter.get());
+      }
+      assertEquals(3, counter.get());
```

Comment:
```suggestion
      assertEquals(1, counter.get());
      for (int i = 1; i < 3; i++) {
        scheduler.run();
        assertEquals(i, counter.get());
        assertTrue(scheduler.isScheduled(group), "Expected group to be scheduled with i = " + i);
      }
      assertEquals(3, counter.get());
      assertFalse(scheduler.isScheduled(group));
```
This way, if `repeatedly()` fails, we get a relatively nice error message right away instead of waiting for the test to reach the time limit. This also makes it a little clearer what's going on.

Reply:
This is missing a `scheduler.run()` on line 408 before the checks making it fail, so I will add that and then commit this.

Reply:
committed, see 6e88f89